README.md

```diff
@@ -1,8 +1,12 @@
 # Elastic Cloud Control
 
-`ecctl` is the CLI on steroids for the Elastic Cloud API. It wraps common operations commonly needed by Elastic Cloud operators within a single command line tool.
+NOTE: **`ecctl` is still in beta and should be considered experimental!**
 
-The goal of this project is to provide an easier way to interact with the Elastic Cloud API, ensuring each one of the provided commands is thoroughly tested.
+`ecctl` is the CLI on steroids for the Elastic Cloud Enterprise API. It wraps typical operations commonly needed by ECE operators within a single command line tool.
+
+The goal of this project is to provide an easier way to interact with the ECE API, ensuring each one of the provided commands is thoroughly tested.
+
+Note that, right now, `ecctl` targets ECE installations only. There are, however, plans to support public EC API in the future.
```

Comment:
I won't use the `EC` abbreviation but rather use `Elasticsearch Service` mostly since it's the first mention and also this is how we refer to our hosted ES service publicly. Elastic Cloud is an umbrella term for Elasticsearch Service, Elastic App Search and Elastic Site Search. See https://www.elastic.co/pricing